src/components/Table/SortableTable.js

```diff
@@ -74,6 +74,7 @@ function defaultRenderRow(
 
 function getSelectableCell(row, rowSelected, onSelect) {
   const selectRowId = uniqueId('select-row-');
+  const rowSelectable = Object.hasOwn(row, 'selectable') ? row.selectable : true;
```

Comment:
Does TS not complain about `Object.hasOwn`? Every time I use it in property, it complains because it's a newish API and the built in TS types don't know about it.

Reply:
Well this is a JS file. So no, but it complained when I used `row.hasOwnProperty('selectable')` so I figured this was the more appropriate approach. Is there another solution to this?

Reply:
`Object.hasOwn` is the best solution but requires the built-in lib.d.ts to be es2022 or greater and doesn't have type guards yet.

You could do `'selectable' in row. That searches the object's prototype chain so it's not the best, but works fine in this case.

You could also change `row.selectable` to `row.disabled` right? That would get rid of this code.

Reply:
`selectable` is just an attribute on the `row` object. It wouldn't matter whether I used `selectable` or `disabled` 

So I think we'd still need the logic here to ensure that either key existed on the object. 

Because if the key didn't exist, it would default to `undefined` a falsey value, therefore every row would be disabled. 

Is the `Object.hasOwn` vs the `'selectable' in row` a blocker? Would the es2022 ever become an issue going forward? 

If not I'll merge as is.

Reply:
> Because if the key didn't exist, it would default to undefined a falsey value

`disabled` is the reverse of `selectable` so you shouldn't need logic to detect if the key is present for `disabled`. `disabled={undefined}` should behave the same as `disabled={false}`.

> Is the Object.hasOwn vs the 'selectable' in row a blocker?

I don't think it's a blocker but it could be if an app was not using lib.d.ts es2022 or greater and did not [`skipLibCheck`](https://www.typescriptlang.org/tsconfig#skipLibCheck).

Reply:
ah, i see what you're saying now. Okay I'll see about changing that 👍

Reply:
Changed it to `disabled` from `selectable` 👍